tutorial/02-the-mercury-type-system.polytex

```diff
@@ -554,8 +549,7 @@ The two declarations could be combined thus
 However, having separate #pred# and #mode# declarations 
 highlights that the \emph{type} of #filter#'s higher order argument
 is written #pred(T)#.  (Separate #mode# declarations are examined in more
-detail in Chapter \XXX{}.)  \XXX{Do I need to explain the parameterised \texttt{in}
-mode?}
+detail in Chapter \XXX{}.)
```

Comment:
What's the reason to remove this? The parameterised `in` mode is the one that the `pred(T)` argument has. Is this already explained earlier?

Reply:
> Is this already explained earlier?

Not explained, but the previous sentence is "Separate mode declarations are examined in more detail in Chapter XXX)". And because it's detailed in the next chapter, I don't think it's worth repeating here.

Reply:
Good enough for me :-)